pubtools/_OIIInspector/_OIIInspector_calls.py

```diff
@@ -0,0 +1,57 @@
+from utils.py import setup_arg_parser
```

Comment:
Could you change the directory structure so that OIIInspector is the base directory? This tool doesn't belong to 'pubtools' family of projects (also, pls remove the underscore)